app/models/hyrax/administrative_set.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Hyrax
+  ##
+  # Valkyrie model for Admin Set domain objects.
+  class AdministrativeSet < Hyrax::Resource
+    attribute :alt_title,   Valkyrie::Types::Set.of(Valkyrie::Types::String)
+    attribute :creator,     Valkyrie::Types::Set.of(Valkyrie::Types::String)
+    attribute :description, Valkyrie::Types::Set.of(Valkyrie::Types::String)
+    attribute :title,       Valkyrie::Types::Set.of(Valkyrie::Types::String)
+  end
+end
```

Comment:
Doesn't it need the following attributes:

date_created: Valkyrie::Types::DateTime,
date_modified: Valkyrie::Types::DateTime,

Reply:
oh, probably!

I can come back review the current adminset models a bit more thoroughly.

Reply:
Turns out `AdminSet` doesn't provide a Hyrax managed `date_created` or `date_modified`.

I think there is more work to do on AdminSet models (representative and thumbnail, definitely). Should we evaluate this as-is and see about getting the rest done on a future Pull request?

Reply:
👍